src/script/assets/AssetCrypto.js

```diff
@@ -17,21 +17,18 @@
  *
  */
 
-window.z = window.z || {};
-window.z.assets = z.assets || {};
-
-z.assets.AssetCrypto = (() => {
+class AssetCrypto {
```

Comment:
Why is this a class? As far as I can tell there are just static methods in there which could be exported by itself.